multiples_3_5/multiples_3_5.rb

```diff
@@ -0,0 +1,4 @@
+def solution(number)
+  list = *(1...number).select { |n| (n % 3).zero? || (n % 5).zero? }
```

Comment:
1. the splat operator (`*`) is usually used to split an array into a list of arguments: 
(ex: for `list(a,b,c)` → `list(*[1,2,3])`), it is not common to use it like it used in this case.  

2. we could omit the splat operator `*` since `select` works on `range`.
  `*(1...number).select` →  `(1...number).select`.
3. `nitpick` in my opinion it is a little more clear to work with number in case of numeric operation and not call a helper method.  
  `{ |n| (n % 3).zero? || (n % 5).zero? }` →  `{ |n| n % 3 == 0 || n % 5 == 0 }`
4. probably it is not necessary to have a separate value `list` since the whole operation could be applied in one line and it is readable. 
5. since we assume that the `range` is always present, we could apply small syntactic sugar: 
  `.inject(0, :+)` → `.inject(:+)`

so the final solution could look like: 

```ruby
(1...number).select { |n| n % 3 == 0 || n % 5 == 0 }.inject(:+)
```